src/public/views/changelist.njk

```diff
@@ -16,7 +16,16 @@
     <button class="btn btn-primary">Add {{ metadata.name | lower }}</button>
   </a>
 </div>
+
 <div class="admin-content">
+  {% if adminEntity.searchFields %}
+  <form method="GET">
+    <div class="search-bar">
+      <input type="text" class="form-control col-5 search-box" name="searchString" id="search" placeholder="Search">
```

Comment:
a `.col-*` should be inside a `.row`. What you probably want is: `.row > .col-12 > .search-bar > .row > .col6 > .searchbox > input + button`, maybe flexing on .searchbox to align the button and input